app/src/main/java/com/willowtree/vocable/presets/CategoriesFragment.kt

```diff
@@ -55,7 +55,7 @@ class CategoriesFragment : BaseFragment() {
                 action = {
                     viewModel.onCategorySelected(category)
                 }
-                if (!isTablet && index + 1 == maxCategories) {
+                if (!isTablet && maxCategories != 1 && index + 1 == maxCategories) {
```

Comment:
Why 1? Worth making a constant to avoid "magic number"s?

Reply:
Yeah it was removing the start margin if there's only one category showing since the next part of the if-block is true. We can clean this up a bit to make it more clear.